scripts/docker/debian/buster/Dockerfile

```diff
@@ -117,18 +116,19 @@ RUN useradd \
     --shell "/bin/bash" \
     jenkins
 
-# install gradle via sdkman (part 1/2) - update shell as root
-RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+# download and install gradle
+RUN cd /tmp && wget https://services.gradle.org/distributions/gradle-6.8.3-bin.zip && unzip gradle-6.8.3-bin.zip && rm gradle-6.8.3-bin.zip && mv gradle-6.8.3 /opt/gradle
```

Comment:
Is gradle 4.4.1-6 (which is in buster) not working?

Reply:
IMHO no. Gradle 4.4 is ages old and there have been multiple potentially breaking changes since then.
Since the main purpose of my contribution is to bring the Java bindings up to scratch, i would rather refrain from using an outdated tool chain.